test/evergreen.yml

```diff
@@ -590,22 +593,7 @@ functions:
         script: |
           set -o errexit
           set -o verbose
-          # Parse through any failures inside the WT_TEST directory.
-          ${python_binary|python3} ../test/evergreen/print_stack_trace.py --unit_test -e ${python_binary|python3} -c WT_TEST -l .
-
-          # Look through the stress format directory for any coredumps.
-          ${python_binary|python3} ../test/evergreen/print_stack_trace.py --format -e ./test/format/t -c test/format/. -l test/format/.
-    - command: s3.put
```

Comment:
Are we still uploading the stacktraces txt?

Reply:
Nope, see [this](https://github.com/wiredtiger/wiredtiger/pull/8377#discussion_r1008989155).